packages/cfd/src/Components/__tests__/cfd-poa.spec.js

```diff
@@ -0,0 +1,296 @@
+import { fireEvent, render, screen, waitFor, waitForElementToBeRemoved } from '@testing-library/react';
+import React from 'react';
```

Comment:
Hello @thisyahlen-deriv,

Shouldn't this be on the first line, since we're still using react v16?